### PR TITLE
Span new child

### PR DESCRIPTION
**Pull request by konstantin-stepanov**
implementation of #83

### CODE REVIEW

File: aiozipkin/span.py [hunk 1]
```diff
@@ -66,6 +66,10 @@ def kind(self: T, span_kind: str) -> T:
     def name(self: T, span_name: str) -> T:
         pass  # pragma: no cover
 
+    @abstractmethod
+    def new_child(self: T, name: str = None, kind: str = None) -> T:
```
Comment (jettify) [inline aiozipkin/span.py]: should be OptStrt or Optional[str], since name or kind could be None

File: aiozipkin/span.py [hunk 2]
```diff
@@ -117,6 +121,9 @@ def kind(self, span_kind: str) -> 'NoopSpan':
     def name(self, span_name: str) -> 'NoopSpan':
         return self
 
+    def new_child(self, name: str = None, kind: str = None) -> 'NoopSpan':
+        return self
```
Comment (jettify) [inline aiozipkin/span.py]: to be on safe side lets create new NoopSpan here

File: aiozipkin/span.py [hunk 3]
```diff
@@ -177,3 +184,11 @@ def kind(self, span_kind: str) -> 'Span':
     def name(self, span_name: str) -> 'Span':
         self._record.name(span_name)
         return self
+
+    def new_child(self, name: str = None, kind: str = None) -> 'Span':
```
Comment (jettify) [inline aiozipkin/span.py]: what if we move this method to SpanAbc? As result we do not need implement it in base classes.

Comment (konstantin-stepanov) [reply aiozipkin/span.py]: I thought about it. But I did it for type hinting of real return classes

File: aiozipkin/span.py (hunk 1 above)
Comment (konstantin-stepanov) [reply aiozipkin/span.py]: done

File: aiozipkin/span.py (hunk 2 above)
Comment (konstantin-stepanov) [reply aiozipkin/span.py]: done... maybe

**Comment by jettify** [comment]
thanks a lot!

**Pull request merged**